assets/queries/cloudFormation/aws/iam_password_without_lowercase_letter/metadata.json

```
@@ -3,7 +3,7 @@
   "queryName": "IAM Password Without Lowercase Letter",
   "severity": "MEDIUM",
   "category": "Best Practices",
-  "descriptionText": "IAM user resource Login Profile Password should have lowercase letter",
+  "descriptionText": "Check if IAM account password has at least one lowercase letter",
```

For this type of query, I suggest something like:

```suggestion
  "descriptionText": "IAM Password should have at least one lowercase letter",
```

(The same applies for the similar queries, like `assets/queries/cloudFormation/aws/iam_password_without_minimum_length`)